traveler_app/src/main/java/com/guestlogix/traveler/settings/TravelerDataStore.java

```diff
@@ -0,0 +1,22 @@
+package com.guestlogix.traveler.settings;
+
+import android.content.Context;
+
+import androidx.annotation.Nullable;
+import androidx.preference.PreferenceDataStore;
+
+import com.guestlogix.travelercorekit.utilities.TravelerPrefs;
+
+public class TravelerDataStore extends PreferenceDataStore {
```

Comment:
whats the purpose of this class?

Reply:
it seems like its just a proxy?

Reply:
by default all settings are saved in the default sharedprefs. this way, we can store it in `TRAVELER_PREFS`

Reply:
Whats the difference between just using TravelePrefs directly then?

Reply:
All of the settings are handled by the `PreferenceManager`. I have to set the data store to `TravelerDataStore` to store our settings in `TRAVELER_PREFS`